README.md

```diff
@@ -59,6 +59,32 @@ make build
 This command executes the Scarb build process, resulting in the creation of a
 Sierra program.
 
+## Test Project
+
+Run the following command to run all tests:
+
+```bash
+make test
+```
+
+If you wish to run tests from a specific module, use the MOD parameter. For instance, to run tests within the Mint module, execute:
+
+```bash
+make test MOD=Mint
+```
+
+To focus on a specific test function, utilize the FN parameter. For instance, to run only the "test_invalid_max_tick" function:
+
+```bash
+make test FN=test_invalid_max_tick
+```
+
+For more precise testing, you can combine both parameters. Remember, these parameters are case-sensitive.
+
+```bash
+make test MOD=Mint FN=test_invalid_max_tick
```

Comment:
This test doesn't work. I get the following error:
`

`
Also there is no` test_invalid_max_tick` function in the Mint module.